### PR TITLE
Fix typos with codespell

### PR DESCRIPTION
**Pull request by cclauss**
https://pypi.org/project/codespell

`codespell --ignore-words-list=bu,extint,fo,ure --skip="./.*,*.json,*.log,*.po,*.svg" --write-changes`

### CODE REVIEW

**Comment by carlosperate** [comment]
Thanks @cclauss!
Is it possible you are using a different version of black? There are quite a lot of formatting changes in this PR.

**Comment by cclauss** [comment]
This commit should be better.

File: mu/modes/api/lego.py [hunk 1]
```diff
@@ -29,10 +29,10 @@
...
...y.info() \nReturns a dict with the Battery subsystem information. Avoid calling\nthis function every time you need a specific item. It’s better to save\nthe result in a local variable and then reference the dict elements as\nneeded:\n\nerror_state - A list of currently active errors incl. 0 (flipper.BATTERY_NO_ERROR) if no errors are active. Possible error are:\n\n\nbattery.BATTERY_NO_ERROR = 0\nbattery.BATTERY_HUB_TEMPERATURE_CRITICAL_OUT_OF_RANGE = -1\nbattery.BATTERY_TEMPERATURE_OUT_OF_RANGE = -2\nbattery.BATTERY_TEMPERATURE_SENSOR_FAIL = -3\nbattery.BATTERY_BAD_BATTERY = -4\nbattery.BATTERY_VOLTAGE_TOO_LOW = -5\n\n\n\ncharger_state - The state of the charter circuit:\n\n\nbattery.DISCHARGING = 0\nbattery.CHARGING_ONGOING = 1\nbattery.CHARGING_COMPLETED = 2\nbattery.FAIL = -1\n\n\n\ncharge_voltage - The battery volgate in mV\n\ncharge_voltage_filtered - The battery volgate (filtered) in mV\n\ntemperature - The battery temperature in degree Celsius.\n\ncharge_current - The battery charging current in mA (the current flowing into the battery)\n\n\nNote\nThe returned value is not calibrated.\n\n\n\nbattery_capacity_left - The battery capacity left in percent of full capacity\n\n\n"
```
Comment (cclauss) [inline mu/modes/api/lego.py:32]: The change here is `Celsius` but GitHub diff seems to be failing to properly highlight the change.

File: mu/contrib/esptool.py
Comment (carlosperate) [inline mu/contrib/esptool.py:1]: esptool is a third party module that we are bringing into the `contrib` folder, so we don't generally touch the code in this file. Would it be okay to revert the fixes to this file? Thanks!

File: mu/modes/api/lego.py (hunk 1 above)
Comment (carlosperate) [reply mu/modes/api/lego.py:32]: Wow, looks like you've found a bug in Github itself!

File: mu/contrib/esptool.py
Comment (cclauss) [reply mu/contrib/esptool.py:1]: `git checkout master -- mu/contrib/esptool.py`

**Comment by cclauss** [comment]
Why are the Python 3.5 tests suddenly failing?

**Comment by cclauss** [comment]
@carlosperate Can you please rerun the one failing test?

**Review by carlosperate — APPROVED** [review]
All good 👍 
Thanks @cclauss!

**Pull request merged**